test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/ODataUriParserTests.cs

```diff
@@ -1183,6 +1183,94 @@ public void ParseComposableEscapeFunctionUrlAndPropertyReturnsTheSameODataPath()
             Assert.Equal("Name", proSegment.Property.Name);
         }
 
+        [Theory]
+        [InlineData("/entitySetEscaped/32:/photos%2F2018%2F%2fFebruary:/Name", "/entitySetEscaped/32/NS.ComposableFunction(arg='photos%2F2018%2F%2fFebruary')/Name", true)]

```

Comment:
>%2F2018%2F%2 [](start = 49, length = 12)

does unescaped "/" work as well? or Do we only allowe customer to use "escaped"?

Reply:
Yes, it would work as well for escaped functions.

---
In reply to: [367060428](https://github.com/OData/odata.net/pull/1621#discussion_r367060428) [](ancestors = 367060428)